packages/server/lib/socket-base.ts

```diff
@@ -478,6 +478,8 @@ export class SocketBase {
                 return (telemetry.exporter() as OTLPTraceExporterCloud)?.send(args[0], () => {}, (err) => {
                   debug('error exporting telemetry data from browser %s', err)
                 })
+              case 'close:extra:targets':
```

Comment:
Actually, why not reset this in `reset:cached:test:state` when all other aspects are cleaned up between tests?

Reply:
It looks like `reset:cached:test:state` only [runs when a new spec is selected](https://github.com/cypress-io/cypress/blob/ab60af86332a2a1c3c65e965dd539f081a28807a/packages/app/src/runner/event-manager.ts#L852-L855) and not between tests.